### PR TITLE
icu: Minor cleanup post merge.

### PR DESCRIPTION
**Pull request by jakirkham**

### CODE REVIEW

File: recipes/icu/meta.yaml [hunk 1]
```diff
@@ -35,20 +36,23 @@ test:
...
-    - pkgdata --help          # [unix]
-    - uconv --help            # [unix]
+
+    # CLI tests
+    - derb --help              # [osx]
+    - genbrk --help            # [unix]
+    - gencfu --help            # [unix]
+    - gencnval --help          # [unix]
+    - gendict --help           # [unix]
+    - genrb --help             # [unix]
```
Comment (jakirkham) [inline recipes/icu/meta.yaml:49]: Fails after this command. Not sure why though as the next command seems to work fine outside of the test step.

File: recipes/icu/meta.yaml [hunk 2]
```diff
@@ -35,20 +36,23 @@ test:
...
+    - icuinfo --help           # [unix]
+    - makeconv --help          # [unix]
```
Comment (jakirkham) [inline recipes/icu/meta.yaml]: Looks like this one has an exit code of 1.

File: recipes/icu/meta.yaml [hunk 3]
```diff
@@ -35,20 +36,21 @@ test:
...
+
+    # CLI tests
+    - genbrk --help            # [unix]
+    - gencfu --help            # [unix]
+    - gencnval --help          # [unix]
+    - gendict --help           # [unix]
+    - genrb --help             # [unix]
+    - icuinfo --help           # [unix]
+    #- makeconv --help          # [unix]
+    #- pkgdata --help           # [unix]
```
Comment (jakirkham) [inline recipes/icu/meta.yaml:52]: These ones have exit code 1 it seems.

**Comment by msarahan** [comment]
Thanks! Merging. I need to add checks for one or two more libraries that the io stuff adds. Will do that in the other PR.

**Pull request merged**